packages/cozy-harvest-lib/src/components/AccountModalWithoutTabs/AccountModalHeader.jsx

```diff
@@ -1,7 +1,9 @@
 import PropTypes from 'prop-types'
 import React from 'react'
+import { useMatch } from 'react-router-dom'
```

Comment:
There is another Header component for V4 (`packages/cozy-harvest-lib/src/components/AccountModalWithoutTabs/ForV4Router/AccountModalWithoutTabs.jsx`)